src/isomorphic/__tests__/ReactPerf-test.js

```diff
@@ -208,6 +215,32 @@ describe('ReactPerf', function() {
     });
   });
 
+  it('should not count replacing null with a native as waste', function() {
+    var element = null;
```

Comment:
Ha, I should’ve realized this before I wrote hundreds of tests using this pattern 😄 .